modules/akka-http/build.sbt

```
@@ -0,0 +1,16 @@
+name := "pureconfig-akka-http"
+
+crossScalaVersions ~= { _.filterNot(_.startsWith("2.11")) }
+
+libraryDependencies ++= Seq(
+  "com.typesafe.akka" %% "akka-stream" % "2.5.31",
```

I see. Nonetheless, this being a library project that does not depend explicitly on any code from `akka-stream`, it's still important that we mark `akka-stream` as `provided` in order to make sure we don't bind client code to a specific version (and force them to provide their own akka-stream version as described [here](https://doc.akka.io/docs/akka-http/current/compatibility-guidelines.html#compatibility-with-akka)).
```suggestion
  "com.typesafe.akka" %% "akka-stream" % "2.5.31" % Provided,
```

---

hmm, didn't think of that. Sounds good. Thanks for the suggestion

---

I forgot about tut, which still needs it to run code in the documentation. I took the liberty of adding tut to the dependency scopes myself, hope it's ok :)